### PR TITLE
added RR, reorged prereg section

### PR DESCRIPTION
**Pull request by poldrack**

### CODE REVIEW

**Comment by poldrack** [comment]
@demidenm please review

File: labguide/research/preregistration.md [hunk 1]
```diff
@@ -59,3 +60,12 @@ pre-registration should specify:
    - Publications should include
     an explicit “Deviations from pre-registration” section that outlines
     any deviations and their rationale.
+
+### Registered Reports
+
+Whenever possible, we support the submission of Registered Reports.
+
+Registered reports are a publication mechanism in which one submits a manuscript that contains an Introduction and Methods section that describe a study prior to execution of the research; it is essential a written version of a pre-registration.  The manuscript is reviewed on the strength of the rationale and methods, and if deemed sufficient it is given an "in-principle acceptance" such that the journal guarantees to publish the paper regardless of the results, as long as the approved methods were followed.
```
Comment (demidenm) [inline labguide/research/preregistration.md:68]: essential --> essentially*. 

guarantees --> agrees*. (some journals are POS and may not)

**Comment by demidenm** [comment]
outside of a couple minor things, looks good to me.

File: labguide/research/preregistration.md (hunk 1 above)
Comment (emdupre) [reply labguide/research/preregistration.md:68]: New line for every sentence ? 🙏

**Pull request merged**